kurtosis/Makefile

```diff
@@ -4,34 +4,11 @@
 ###                                Kurtosis                                 ###
 ###############################################################################
 
-# Installs Kurtosis if not already installed
-install-kurtosis:
-	@echo "Checking for Kurtosis installation..."
-	@if ! command -v kurtosis &> /dev/null; then \
-		echo "Kurtosis could not be found, installing..."; \
-		OS=$$(uname -s | tr A-Z a-z); \
-		if [ "$$OS" = "darwin" ]; then \
-			brew install kurtosis-tech/tap/kurtosis-cli; \
-		elif [ "$$OS" = "linux" ]; then \
-			ARCH=$$(uname -m); \
-			if [ "$$ARCH" = "x86_64" ]; then ARCH="amd64"; \
-			elif [ "$$ARCH" = "arm64" ]; then ARCH="arm64"; \
-			else echo "Unsupported architecture $$ARCH for Kurtosis installation" && exit 1; fi; \
-			TAG=`curl -s "https://api.github.com/repos/kurtosis-tech/kurtosis-cli-release-artifacts/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/'`; \
-			curl -Lo kurtosis.tar.gz "https://github.com/kurtosis-tech/kurtosis-cli-release-artifacts/releases/download/$TAG/kurtosis-cli_${TAG}_${OS}_${ARCH}.tar.gz"; \
-			tar -xzf kurtosis.tar.gz; \
-			rm kurtosis.tar.gz; \
-			chmod +x kurtosis; \
-			sudo mv kurtosis /usr/local/bin/; \
-		else \
-			echo "Unsupported OS $$OS for Kurtosis installation" && exit 1; \
-		fi; \
-	else \
-		echo "Kurtosis is already installed"; \
-	fi
```

Comment:
I mentioned during planning today about being fine with fixing the Ubuntu compatibility issue and keeping this install code. I changed my mind as this blob of code is complex and difficult to test and guarantee it works on different platforms. I think we should defer the installation to the user as a prerequisite, its already mentioned in the kurtosis/README.

Reply:
if we collectively decide we wanna get rid of the install code and let the user ensure kurtosis is installed before spinning up the local network, I am okay @abi87 @calbera wanted to know your thoughts